test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestPrepareResult.java

```diff
@@ -0,0 +1,8 @@
+package io.quarkus.test.junit;
+
+import io.quarkus.bootstrap.app.AugmentAction;
+import io.quarkus.bootstrap.app.CuratedApplication;
+
+record QuarkusTestPrepareResult(AugmentAction augmentAction, QuarkusTestProfile profileInstance,
```

Comment:
I don't love the name of this, just because it's so long. I wondered about dropping `Quarkus` since it doesn't add much information (of *course* it's something to do with Quarkus), but then I realised maybe it's supposed to be Quarkus as in `QuarkusTest`. But `QuarkusTest` isn't the right qualifier, because `QuarkusMainTest` also uses it. So I'd vote for sticking with `PrepareResult`. It's not the best name ever, but it's familiar to maintainers, and the signal to noise is slightly higher than `QuarkusTestPrepareResult`.